.mergify.yml

```diff
@@ -49,3 +49,8 @@ pull_request_rules:
         branches:
           - "7.12.x"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: delete head branch after merge
+    conditions:
+      - merged
+    actions:
+      delete_head_branch:
```

Comment:
I guess, the PRs will never be created from any upstream branches, right? If so, +1

Reply:
Yes, I think that's something `mergify` does under the hood for the backport action.

I only configured for the specific bump automation we have in place in other repos.